mjx/pyproject.toml

```diff
@@ -44,4 +44,4 @@ mjx-viewer = "mujoco.mjx.viewer:main"
 Homepage = "https://github.com/google-deepmind/mujoco/tree/main/mjx"
 Documentation = "https://mujoco.readthedocs.io/en/3.1.5"
 Repository = "https://github.com/google-deepmind/mujoco/tree/main/mjx"
-Changelog = "https://mujoco.readthedocs.io/en/3.1.5/changelog.html"
+Changelog = "https://mujoco.readthedocs.io/en/3.1.5/changelog.html"
```

Comment:
I suspect this is an unneeded change - can you revert?  Otherwise, let me know what this is for.